etcdutl/etcdutl/snapshot_command.go

```diff
@@ -110,6 +112,7 @@ func SnapshotRestoreCommandFunc(restoreCluster string,
 	restorePeerURLs string,
 	restoreName string,
 	skipHashCheck bool,
+	ignoreBootstrapVerify bool,
```

Comment:
maybe call it `ignoreExternalVerification` to stress that we want to skip the 'not hermetic' verifications, that e.g. depends on DNS.